hw/ip/keymgr/data/keymgr_testplan.hjson

```diff
@@ -27,10 +27,30 @@
             - Check interrupts `op_done` is triggered when operation is done.
             - Check `err` and alert `operation_err` are triggered after invalid operation.
             - Check KMAC key, KMAC data and output SW data for correctness.
+            - For invalid operations, check KMAC key, KMAC data and output SW data don't match to
```

Comment:
nit: Sounds like an overkill for a smoke test (smoke implies basic testing). You can perhaps split this into another entry. Though, I don't feel strongly about this so I am fine either way.

Reply:
Since I have moved all the randomization of key and data to the `random` test, this is just to do 3 operations in all states. In non-working states, operation becomes invalid and checking is done in scb. I prefer to the invalid operations rather than splitting them to another test.